defaults/main.yml

```diff
@@ -3,12 +3,26 @@ unbound_anchorfile: "/var/lib/unbound/root.key"
 unbound_anchor_enabled: true
 unbound_logfile: "/var/log/unbound.log"
 unbound_configuration:
-    - verbosity: 1
-    - do-ip4: "yes"
-    - do-ip6: "no"
-    - num-threads: 1
-    - pidfile: "/var/run/unbound.pid"
-    - logfile: "{{unbound_logfile}}"
+  - name: verbosity
+    value: "1"
```

Comment:
@thalunil this syntax is little more complicated, it's not possible to keep the old one ?
(we lost the compatility also)

Reply:
It is possible by changing the vars and the jinja2 template: 

vars:
    unbound_configuration:
      verbosity: "2"
      do-ip4: "yes"
      do-ip6: "yes"
      num-threads: "1"

01general.conf.j2:

server:
{% for key,value in unbound_configuration.items() %}
    {{ key }}: {{ value }}
{% endfor %}

Reply:
in 10zone.conf.j2 change iteritems() to items(). I'll create a pull request if you want and if this issue is still open.

Reply:
@rogierm thanks, @thalunil looks inactive on his PR
I will be able to merge if you can remove "do-daemonize" https://github.com/jdauphant/ansible-role-unbound/pull/9/files#r210898409 and restore the check https://github.com/jdauphant/ansible-role-unbound/pull/9/files#r210898567